### ISSUE TITLE
Export common ABIs which are already in source code

### ISSUE THREAD

**Issue by itsmnthn** [ThalaLabs/surf#178]
- I'm trying surf for the first time and it is unclear as beginner to how to use [aptos-wallet-adapter](https://github.com/aptos-labs/aptos-wallet-adapter) with surf?
- Also I see COIN_ABI in source code and used in examples in readme but how do I import them to use? I know it is not gud idea to include ABIs with this but I think common ABIs can be exported along with this package to make it easy to get started for common features like fetching balances and transferring tokens.

**Comment by itsmnthn** [comment]
In example there is transfer example which uses account from private key but that will not be the case in production dapp, its going to be signer(account) from connected wallet.

It would be helpful to have such example.

**Comment by SamuelQZQ** [comment]
@itsmnthn please look at this file about how to use with wallet: https://github.com/ThalaLabs/surf/blob/main/example/app/page.tsx

**Comment by SamuelQZQ** [comment]
@itsmnthn I agree, we should export common ABIs from this package. It's also mentioned in #8. I will add it, maybe this week.

**Comment by SamuelQZQ** [comment]
https://github.com/ThalaLabs/surf/pull/231
